srcpkgs/perl-Text-ASCIITable/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'perl-Text-ASCIITable'
+pkgname=perl-Text-ASCIITable
+version=0.22
+revision=2
+noarch=yes
+wrksrc="${pkgname/perl-/}-${version}"
+build_style=perl-ModuleBuild
+hostmakedepends="perl perl-Module-Build"
+makedepends="perl"
+depends="${makedepends}"
```

Comment:
Might be a matter of taste, but if it only depends on perl I think using `${makedepends}` here looks a bit ugly. Just `depends="perl"` suffices.